src/go/configmanager/flags/flags.go

```diff
@@ -161,6 +161,7 @@ var (
 	BackendRetryNum = flag.Uint("backend_retry_num", 1,
 		`The allowed number of retries. Must be >= 0 and defaults to 1. This retry
 	setting will be applied to all the backends if you have multiple ones.`)
+	BackendPerTryTimeoutSec = flag.Uint("backend_per_try_timeout_sec", 0, "The upstream timeout per retry attempt in second. If unspecified, a request that times out won't be retried.")
```

Comment:
what does 0 mean?  should it be "None"?

Reply:
golang is strong typing so it won't accept nil for primitive uint type. 0 here means the default value.

Not sure if I get your point.

Reply:
maybe state that 0 means "nospecified"?

Reply:
Done.